src/components/shell/shell.js

```diff
@@ -273,7 +273,10 @@ class Shell extends BaseComponent {
    * @memberof Shell
    */
   componentDidUpdate(prevProps: Object): void {
-    if (!this.props.prePlayback && prevProps.prePlayback) {
+    // Update the hover state if the transition was from pre playback screen
+    // or after an ad break
+    if (!this.props.prePlayback && prevProps.prePlayback ||
+      !this.props.adBreak && prevProps.adBreak) {
```

Comment:
separate the conditions with `()`:
```
if ((!this.props.prePlayback && prevProps.prePlayback) ||
   (!this.props.adBreak && prevProps.adBreak)) {
```
just for readability, the logic  is correct.